rules/macos/process_creation/proc_creation_macos_binary_padding.yml

```diff
@@ -16,13 +18,18 @@ logsource:
 detection:
     selection1:
         Image|endswith: '/truncate'
-        CommandLine|contains: '-s'
+        CommandLine|contains: '-s +'
+    filter1:
```

Comment:
We are looking for `-s +` and there is only one time the option, so the filter will never happend.

Reply:
Nice catch. The filter is unnecessary in this case then.